src/js/utils/form-utils.jsx

```diff
@@ -79,3 +79,6 @@ renderField.defaultProps = {
   arrayField: false,
   label: '',
 };
+
+export const generateKey = () =>
```

Comment:
I assume this is for the random unique number needed for the stock movement. I don't love that Jesse wants to generate this unique ID BEFORE we save it to the database. But if that's really required, we'll add a REST endpoint for you guys to generate the ID as there's some config / logic that needs to be taken into consideration. For now, this is fine, but start to think about mocking an API here.